tests/e2e/lib/dpa_helpers.go

```diff
@@ -171,14 +193,26 @@ func (v *DpaCustomResource) CreateOrUpdateWithRetries(spec *oadpv1alpha1.DataPro
 	)
 	for i := 0; i < retries; i++ {
 		if cr, err = v.Get(); apierrors.IsNotFound(err) {
```

Comment:
was there repeated errors creating the dpa here or is this just an improvement?

Reply:
Ensuring the content of v.CustomResource that's all.